### ISSUE TITLE
Action hanging forever after error

### ISSUE THREAD

**Issue by szymon-rd** [BaileyJM02/markdown-to-pdf#19]
I get the following logs, and the action just hangs:
```
Built HTML file: final.html
Built PDF file: final.pdf
Gracefully shut down image server.
(node:1) UnhandledPromiseRejectionWarning: TimeoutError: Navigation Timeout Exceeded: 30000ms exceeded
    at /node_modules/puppeteer/lib/LifecycleWatcher.js:142:21
  -- ASYNC --
    at Frame.<anonymous> (/node_modules/puppeteer/lib/helper.js:110:27)
    at Page.goto (/node_modules/puppeteer/lib/Page.js:629:49)
    at Page.<anonymous> (/node_modules/puppeteer/lib/helper.js:111:23)
    at /markdown-to-pdf.js:159:14
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by BaileyJM02** [comment]
Thank you, I will look into this shortly. Are you able to [email me](mailto:bailey@hey.com?subject=\[markdown-to-pdf#19\]%20Example%20file) the `final.md` file for testing?

**Comment by JohnMeyerhoff** [comment]
This should be fixed by https://github.com/BaileyJM02/markdown-to-pdf/pull/22
by adding https://github.com/BaileyJM02/markdown-to-pdf/pull/22/commits/3ff190b5f2bada1818c62359bd3f0658ad31d826
and https://github.com/BaileyJM02/markdown-to-pdf/pull/22/commits/d743c4256a9a979b2ca1b5fb89c266c92dfe0f23

**Comment by szymon-rd** [comment]
> Thank you, I will look into this shortly. Are you able to [email me](mailto:bailey@hey.com?subject=\[markdown-to-pdf#19\]%20Example%20file) the `final.md` file for testing?

It's from here
https://github.com/sszczep/BD2021/tree/main/content
If you merge these MDs with `cat *.md > final.md`, just like in the gh action (we used pandoc instead)

**Comment by BaileyJM02** [comment]
Closing as this was fixed in #22.
